### ISSUE TITLE
serverUuid should follow immediately after the equal sign

### ISSUE THREAD

**Issue by aristosv** [linuxserver/docker-plex#128]
I used this command to install plex inside a docker container running on Debian 9.3

`docker create --name=plex -h PlexMediaServer --net=host -e VERSION=latest -e PUID=1001 -e PGID=1001 -e TZ=Asia/Nicosia -v /mnt/containers/plex/config:/config -v /mnt/containers/plex/tvshows:/tvshows -v /mnt/containers/plex/movies:/movies -v /mnt/containers/plex/transcode:/transcode linuxserver/plex`

This is the log output
https://pastebin.com/seraVtDG

Basically there's an error saying 

Error in command line:the argument for option '--serverUuid' should follow immediately after the equal sign

**Comment by aptalca** [comment]
That's usually a harmless message in the log, happens until you login and register your server with your account. Your server may be crashing for a different reason. Check the plex media server log in the config folder.

**Comment by aristosv** [comment]
But the web interface http://192.168.1.41:34200/web is not avaialble

**Comment by aristosv** [comment]
This is the log file. There's a message "The database is locked".
But which database?

```
Mar 10, 2018 16:09:59.975 [0x7ff2bde66700] INFO - Plex Media Server v1.12.0.4829-6de959918 - ubuntu docker x86_64 - build: linux-ubuntu-x86_64 ubuntu - GMT 02:00
Mar 10, 2018 16:09:59.976 [0x7ff2bde66700] INFO - Linux version: 4.9.0-6-amd64 (#1 SMP Debian 4.9.82-1+deb9u3 (2018-03-02)), language: en-US
Mar 10, 2018 16:09:59.976 [0x7ff2bde66700] INFO - Processor Intel(R) Core(TM) i7-4785T CPU @ 2.20GHz
Mar 10, 2018 16:09:59.977 [0x7ff2bde66700] INFO - /usr/lib/plexmediaserver/Plex Media Server
Mar 10, 2018 16:09:59.945 [0x7ff2c9dee800] DEBUG - BPQ: [Idle] -> [Starting]
Mar 10, 2018 16:09:59.954 [0x7ff2c9dee800] DEBUG - Opening 20 database sessions to library (com.plexapp.plugins.library), SQLite 3.13.0, threadsafe=1
Mar 10, 2018 16:10:01.868 [0x7ff2bd665700] DEBUG - Job running: '/usr/lib/plexmediaserver/CrashUploader' '--directory=/config/Library/Application Support/Plex Media Server/Crash Reports/1.12.0.4829-6de959918' '--version=1.12.0.4829-6de959918' '--platform=linux-ubuntu-x86_64 (4.9.0-6-amd64 (#1 SMP Debian 4.9.82-1+deb9u3 (2018-03-02)))' '--serverUuid=' '--userId=NOUSERID' '--url=https://crashreport.plexapp.com'
Mar 10, 2018 16:10:01.884 [0x7ff2bce64700] ERROR - JobManager: child process returned: 1 (Operation not permitted)
Mar 10, 2018 16:10:15.007 [0x7ff2c9dee800] ERROR - SQLITE3:0x10, 5, statement aborts at 1: [PRAGMA journal_mode=WAL] database is locked
Mar 10, 2018 16:10:30.245 [0x7ff2c9dee800] ERROR - SQLITE3:0x10, 5, statement aborts at 1: [PRAGMA journal_mode=WAL] database is locked
Mar 10, 2018 16:10:45.467 [0x7ff2c9dee800] ERROR - SQLITE3:0x10, 5, statement aborts at 1: [PRAGMA journal_mode=WAL] database is locked
Mar 10, 2018 16:11:00.690 [0x7ff2c9dee800] ERROR - SQLITE3:0x10, 5, statement aborts at 1: [PRAGMA journal_mode=WAL] database is locked
Mar 10, 2018 16:11:15.913 [0x7ff2c9dee800] ERROR - SQLITE3:0x10, 5, statement aborts at 1: [PRAGMA journal_mode=WAL] database is locked
Mar 10, 2018 16:11:31.135 [0x7ff2c9dee800] ERROR - SQLITE3:0x10, 5, statement aborts at 1: [PRAGMA journal_mode=WAL] database is locked
Mar 10, 2018 16:11:46.359 [0x7ff2c9dee800] ERROR - SQLITE3:0x10, 5, statement aborts at 1: [PRAGMA journal_mode=WAL] database is locked
Mar 10, 2018 16:11:59.946 [0x7ff2bc663700] DEBUG - BPQ: [Starting] -> [Processing]
Mar 10, 2018 16:12:01.582 [0x7ff2c9dee800] ERROR - SQLITE3:0x10, 5, statement aborts at 1: [PRAGMA journal_mode=WAL] database is locked
Mar 10, 2018 16:12:16.825 [0x7ff2c9dee800] ERROR - SQLITE3:0x10, 5, statement aborts at 1: [PRAGMA journal_mode=WAL] database is locked
Mar 10, 2018 16:12:32.049 [0x7ff2c9dee800] ERROR - SQLITE3:0x10, 5, statement aborts at 1: [PRAGMA journal_mode=WAL] database is locked
Mar 10, 2018 16:12:47.272 [0x7ff2c9dee800] ERROR - SQLITE3:0x10, 5, statement aborts at 1: [PRAGMA journal_mode=WAL] database is locked
```

**Comment by aptalca** [comment]
Plex database. Search on plex forums. It's a common issue. Whenever something else accesses the plex database (an app like plexemail or the user via sqlite client) plex database gets locked or corrupted. You should be able to restore from a backup. 

If this is a brand new install, then maybe you have your config folder on a network share? That can also cause issues with sqlite databases. Local folder is recommended

**Comment by aristosv** [comment]
You're right, the config folder is on a network share. I wanted to use plex in docker because I like the idea of separating the app from its configuration. But if I can't have the configuration on a network share, away from the server running docker, I don't see the point in using plex in a container.

**Comment by mauvehed** [comment]
I just ran into this same problem. Plex in Docker with /config volume being mounted from a network share (CIFS). Are we saying this is an unsupported method of deployment? That seems, odd..

**Comment by aptalca** [comment]
Correct. Sqlite does not like being on a network share.

You can maybe try things like putting the config folder on a network share but then symlinking the database folder back to a local location. I don't know, maybe that will work. 

Why do you guys need to put the config folder on a network share?
